src/metabase/models/metric.clj

```diff
@@ -65,7 +65,7 @@
 
 ;; ## Persistence Functions
 
-(defn create-metric
+(defn create-metric!
```

Comment:
what's the thinking behind adding the `!` mark here?  doesn't seem like these functions are dangerous in the clojure/STM way that suffix pertains to.